packages/webapp/package.json

```diff
@@ -15,6 +15,7 @@
   "prettier": "@dailydotdev/prettier-config",
   "dependencies": {
     "@dailydotdev/shared": "*",
+    "caniuse-lite": "^1.0.30001239",
```

Comment:
Why did you add this dependency?

Reply:
I prompted from the terminal to run the command.
I will remove that.